scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
+}
+var url = "https://api.stackexchange.com/2.2/search/advanced?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
```

Comment:
You should not be using the **stackexchange API** unless it is public (you can use without API keys ex Github), here you are using *API key* which I don't think is legal (API keys are personal and should not be shared with others).

The whole point of Loklak is to extract data and provide it to users without tracking them. Please get the *HTML* of a page and parse it using library like `cheerio`, then create the required JSONObject.